cmd/draft/draft.go

```diff
@@ -72,8 +83,8 @@ func newRootCmd(out io.Writer, in io.Reader) *cobra.Command {
 		newCreateCmd(out),
 		newHomeCmd(out),
 		newInitCmd(out, in),
-		newUpCmd(out),
-		newVersionCmd(out),
+		addFlagsTLS(newUpCmd(out)),
+		addFlagsTLS(newVersionCmd(out)),
 		newPluginCmd(out),
```

Comment:
Would it be worth sending the flags to plugins? I know they can request a tunnel to draftd (similar to helm plugins requesting a tunnel to tiller), so they may need info to make a proper secure connection to draftd.

Reply:
Possibly, but the TLS added here is primarily to secure grpc communication between draft client and server.